metrics/mixin/alerts/encryption-external.libsonnet

```diff
@@ -0,0 +1,27 @@
+{
```

Comment:
Do we require these changes in external encryption file? Do we enable KMS and external mode together?
@umangachapagain ^^^

Reply:
One of the above comments answer this (we are enabling KMS in internal and external cluster).

Reply:
I mean not together but I don't see it mentioned anywhere that we don't support external mode. So, added an alert for external mode as well.